docs/framework/react/api/router/useMatchHook.md

```diff
@@ -36,11 +36,26 @@ The `useMatch` hook accepts a single argument, an `options` object.
 
 ## Examples
 
+### accessing a route match
+
+```tsx
+import { useMatch } from '@tanstack/react-router'
+
+function Component() {
+  const match = useMatch({ from: '/posts/$postId' })
+  //     ^? strict match for RouteMatch
+  // ...
+}
+```
+
+### accessing the root route's match
+
 ```tsx
 import { useMatch } from '@tanstack/react-router'
+import { rootRouteId } from '@tanstack/react-router'
```

Comment:
import {
  useMatch,
  rootRouteId // use this token .... 
} from '@tanstack/react-router'
... 
```

Reply:
done